app/main.py

```diff
@@ -1,25 +1,69 @@
+from __future__ import annotations
+from typing import Any
 from abc import ABC
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: Visitor, name: str) -> None:
+        self.name = "_" + name
+
+    def __get__(
+            self,
+            instance: Any,
+            owner: Visitor
+    ) -> Visitor:
+        return getattr(instance, self.name)
+
+    def __set__(self, instance: SlideLimitationValidator, value: int) -> None:
```

Comment:
You could use `Any` for `instance` and `owner` parameters as well in the type annotation